doc/spark_coverage_agg.txt

```diff
@@ -0,0 +1,9 @@
++-------+----------------------------------------------+---+
+|result |reason                                        |cnt|
++-------+----------------------------------------------+---+
+|FAILED |Unsupported                                   |137|
+|FAILED |Failed on native side                         |4  |
+|PASSED |OK                                            |254|
+|SKIPPED|null                                          |12 |
+|FAILED |Failed on something else. Check query manually|10 |
```

Comment:
Hmm, I'm not sure I understand what this document represents?

Reply:
SKIPPED means the indicator, there is no examples in the spark catalog, I'll add more meaningful message